terralab/utils.py

```diff
@@ -27,10 +27,19 @@ def wrapper(*args: str, **kwargs: str) -> Any:
         try:
             return func(*args, **kwargs)
         except ApiException as e:
+            message = "[no message body]"
             if e.body is not None:
-                formatted_message = f"API call failed with status code {e.status} ({e.reason}): {json.loads(e.body)['message']}"
-            else:
-                formatted_message = f"API call failed with status code {e.status} ({e.reason}): [no message body]"
+                message = json.loads(e.body)["message"]
+            if e.status == 401 and message == "User not found":
+                LOGGER.error(
+                    add_blankline_before(
+                        "User not found in Terra. Are you sure you've registered at http://app.terra.bio ?"
```

Comment:
i feel like i know why it exists but the space before the `?` looks weird

Reply:
yeah if there's no space, at least my system thought the question mark was part of the link. does this look better to you?
`User not found in Terra. Are you sure you've registered (http://app.terra.bio)?`

Reply:
what if we did 

```
User not found in Terra. Are you sure you've registered? Visit https://app.terra.bio to register
```

Reply:
that's way better, thanks!